app/views/ontologies/browser/browse.html.haml

```diff
@@ -1,16 +1,14 @@
 .browse-center
   .browse-container
-
-    .browse-submit-new-ontology-and-you-are-admin-container
+    .container.align-alert
       - if session[:user]&.admin?
-        %div
-          %div.browse-you-are-admin
-            %h3 Welcome admin
-            %div
-              %svg{:fill => "none", :height => "11", :viewbox => "0 0 11 11", :width => "11", :xmlns => "http://www.w3.org/2000/svg"}
-                %circle{:cx => "5.5", :cy => "5.5", :fill => 'var(--admin-color)', :r => "5.5"}
-              %p This coloring indicates admin-only features
-
+        = render Display::AlertComponent.new(type: 'info', style: "justify-content: space-between;") do
+          %strong Welcome admin
```

Comment:
you can create here a div to set the `justify-content: center` instead of adding it into the component? (see comment in the alert component )

Reply:
I change it 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/fff2b69d-d337-4ca3-ad19-a0fc85bb6165)